click-spark.js

```diff
@@ -1,7 +1,6 @@
 class ClickSpark extends HTMLElement {
   constructor() {
     super();
-    this.svg;
     this.attachShadow({ mode: "open" });
     this.clickEvent = this.handleClick.bind(this);
```

Comment:
FWIW, this would not be necessary if we [used `handleEvent` instead](https://gomakethings.com/the-handleevent-method-is-the-absolute-best-way-to-handle-events-in-web-components/):

```javascript
  connectedCallback() {
    this.setupSpark();
    this.parentNode.addEventListener("click", this);
  }

  handleEvent(e) {
    this.setSparkPosition(e);
    this.animateSpark();
  }
```

That also comes with a [variety of other advantages](https://web.archive.org/web/20240121164212/https://scribe.rip/webreflection/dom-handleevent-a-cross-platform-standard-since-year-2000-5bf17287fd38#a0ff).